### PR TITLE
DOC-2256 doc(repave): document the new workflow(hostname-based) of replacing a node;

### PR DESCRIPTION
**Pull request by zejunlitg**
Original TP ticket: https://graphsql.atlassian.net/browse/TP-5521
The internal documentation of the new node replacement: https://graphsql.atlassian.net/wiki/spaces/infra/pages/3375235074/Node+replacement+V2

Doc ticket: https://graphsql.atlassian.net/browse/DOC-2256

Screenshot 2024-08-22 at 5 30 21 PM
Screenshot 2024-08-22 at 5 30 29 PM

### CODE REVIEW

File: modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc [hunk 1]
```diff
@@ -2,7 +2,12 @@
 :description: This page describes the procedure to replace a node in a non-ha cluster.
 
 //welcome and introduction
-This guide outlines the procedure for replacing a node in a non-High Availability (HA) cluster. If your system uses xref:ha-overview.adoc[High Availability], refer to the documentation on removing a failed node in xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[].
+This guide outlines the procedure for replacing a node in a cluster regardless of whether it is an High Availability(HA) cluster. If your system uses xref:ha-overview.adoc[High Availability], refer to the documentation on removing a failed node in xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[Removal of Failed Nodes].
```
Comment (chengjie-qin) [inline modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc]: If your system uses HA ** and you do NOT have a replacement node to use**, refer to xxxx

File: modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+
+== Prerequisites
+* TigerGraph is installed with HostName, refer to xref:bare-metal-install.adoc[HostName Installation]. Additionally, TigerGraph should be installed on a device that can be unmounted from the exiting machine and mounted to the new machine.
```
Comment (chengjie-qin) [inline modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc]: existing

File: modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc (hunk 1 above)
Comment (zejunlitg) [reply modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc]: applied

File: modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc (hunk 2 above)
Comment (zejunlitg) [reply modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc]: I meant to express the idea that the node is 'leaving' the cluster hence I called it 'exiting', but looks like it's not straightforward enough so I changed to `node to be replaced` and `to-be-replaced node`.

**Review by chengjie-qin — APPROVED** [review]

**Review by arun-TG-PM — APPROVED** [review]

**Pull request merged**